src/main/java/com/github/terma/jenkins/githubprcoveragestatus/SonarMasterCoverageRepository.java

```diff
@@ -20,27 +22,32 @@
 
     private static final String SONAR_SEARCH_PROJECTS_API_PATH = "/api/projects/index";
     private static final String SONAR_COMPONENT_MEASURE_API_PATH = "/api/measures/component";
-    private static final String SONAR_OVERALL_LINE_COVERAGE_METRIC_NAME = "overall_line_coverage";
+    public static final String SONAR_OVERALL_LINE_COVERAGE_METRIC_NAME = "coverage";
```

Comment:
It works for me(I have something like 80% vs 82% coverage in comments). But I still would prefer to refactor this logic in another PR. 
Because with sonar we need to have the same percentage formula for different parsers. For example, we use cobertura(which currently has a formula (branch_coverage + line_coverage) / 2 and from sonar we use only line_coverage, so the numbers are different although nothing was really changed).